### PR TITLE
Fix bugs & Update player

### LINKED ISSUE #499: something wrong with that game
Screen Shot 2021-02-21 at 6 29 10 PM

### PR DESCRIPTION
**Pull request by EgoriusRus**

### CODE REVIEW

File: components/GameDebugPanel.vue [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="debug-panel">
+    111
```
Comment (kezlya) [inline components/GameDebugPanel.vue]: WTF ?

**Review by kezlya — CHANGES_REQUESTED** [review]
please remove debugging 111 :)

**Review by Klimonov — APPROVED** [review]
LGTM

**Review by kezlya — APPROVED** [review]
LGTM

**Pull request merged**